spec/features/users/sign_in_spec.rb

```diff
@@ -838,13 +838,14 @@
   end
 
   context 'ial2 param on sign up screen' do
-    let(:campaign) { 'campaign1' }
+    let(:ok_campaign) { 'campaign1' }
+    let(:bad_campaign) { 'bad_campaign' }
 
-    before do
-      visit root_path(ial: 2, campaign: campaign)
+    before do |test|
+      visit root_path(ial: 2, campaign: test.metadata[:bad_campaign] ? bad_campaign : ok_campaign)
```

Comment:
I think this could
```ruby
before do
  visit root_path(ial: 2, campaign: campaign)
end
```
and a default
```ruby
let(:campaign) { ok_campaign }
```
```

Reply:
yes i like!